docs/blog/2020-09-15-datadog-apm-in-azure-service-fabric.md

```diff
@@ -0,0 +1,156 @@
+    <ConfigOverrides />
++   <Policies>
++     <ContainerHostPolicies CodePackageRef="Code">
++       <PortBinding ContainerPort="5002" EndpointRef="DatadogTypeEndpoint" />
++       <PortBinding ContainerPort="8126" EndpointRef="DatadogTraceEndpoint" />
++       <PortBinding ContainerPort="8125" EndpointRef="DatadogStatsEndpoint" />
++     </ContainerHostPolicies>
++   </Policies>
+  </ServiceManifestImport>
+```
+
+## Installing the .NET Tracer
+
+The tracer requires machine administrator permissions.
+Add the SetupAdminUser to the Principals section in the ApplicationManifest.xml. If the Principals section is missing, add it.
+
+```
+  <Principals>
+    <Users>
+      <User Name="SetupAdminUser">
+        <MemberOf>
+          <SystemGroup Name="Administrators" />
+        </MemberOf>
+      </User>
+    </Users>
+  </Principals>
+```
+
+Within the ServiceManifestImport of the service responsible for deploying the tracer to the cluster, give the Setup script SetupAdminUser as the executing user.
+
+```diff
+  <ServiceManifestImport>
+    <ServiceManifestRef ServiceManifestName="ServiceThatDeploysDatadogTracerPkg" ServiceManifestVersion="1.0.0" />
+    <ConfigOverrides />
++   <Policies>
++     <RunAsPolicy CodePackageRef="Code" UserRef="SetupAdminUser" EntryPointType="Setup" />
++   </Policies>
+  </ServiceManifestImport>
+```  
+
+Within the ServiceManifest.xml of the service responsible for deploying the Datadog Tracer, add the reference to the install script.
+```diff
+  <CodePackage Name="Code" Version="1.0.0">
++   <SetupEntryPoint>
++     <ExeHost>
++       <Program>DatadogInstall.bat</Program>
++       <WorkingFolder>CodePackage</WorkingFolder>
++     </ExeHost>
++   </SetupEntryPoint>
+    <EntryPoint>
+      <ExeHost>
+        <Program>ServiceThatDeploysDatadogTracer.exe</Program>
+        <WorkingFolder>CodePackage</WorkingFolder>
+      </ExeHost>
+    </EntryPoint>
+  </CodePackage>
+```
+
+Include the DatadogInstall.bat and DatadogInstall.ps1 scripts in the project responsible for deploying the tracer.
+Ensure that in the properties of both scripts, you set that they are copied to the output directory.
+
+![Copy to output directory](https://user-images.githubusercontent.com/1801443/93110062-d05a9480-f682-11ea-8fb4-7b266f576f68.png)
+
+The latest representation of this install process is here: https://github.com/DataDog/azureservicefabric-dotnet-tracing-sample
+ - [Batch Script](https://github.com/DataDog/azureservicefabric-dotnet-tracing-sample/blob/master/VotingWeb/DatadogInstall.bat)
+ - [Powershell Script](https://github.com/DataDog/azureservicefabric-dotnet-tracing-sample/blob/master/VotingWeb/DatadogInstall.ps1)
+
+
+### That's all folks
+---
+
+The next time you deploy this application to the cluster, your application should start sending traces through to Datadog.
+This also enables custom statistics through the DogStatsD client and custom traces through the Datadog.Trace library.
```

Comment:
```suggestion
This also enables custom metrics through DogStatsD and custom traces through the `Datadog.Trace` library.
```

Reply:
We should link to both NuGet packages.

Reply:
Could we please call the directory "blog/" rather than "blog*s*"?

Reply:
Sorry, never mind, confused branch and folder :)